### PR TITLE
Initial commit to create the barebones stats component and stats being

### PR DESCRIPTION
**Pull request by ldalton02**
passed down from landing page.

Adds some css into stats.css to style the bar.

Uses Math.random to generate random numbers from 1 to 10 for initial
stats.

### CODE REVIEW

**Comment by ldalton02** [comment]
closes #22

**Comment by ldalton02** [comment]
Let me know what you think, I can always add more changes/change font if it doesn't look close enough to figma

**Review by nendow02 — APPROVED** [review]
lgtm, I’m going to request @reginawang3495 to quickly look over this and @archishadatta too just because there’s some css in here. Nice job Luke!

File: src/styles/stats.scss [hunk 1]
```diff
@@ -1,12 +1,46 @@
...
+.stats-display {
+  margin: 0;
+}
+
+.stats-display:nth-child(1) {
+  color: #ff5630;
+}
+
+.stats-display:nth-child(2) { 
+  color: #ffab00;
```
Comment (archishadatta) [inline src/styles/stats.scss:27]: Use scss variables for the colors

File: src/styles/stats.scss [hunk 2]
```diff
@@ -1,12 +1,46 @@
 @import 'global';
 
 #bar {
+  align-items: center;
   background-color: #323232;
   border-radius: 10px;
   bottom: 50px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
   height: 50px;
```
Comment (archishadatta) [inline src/styles/stats.scss:10]:
```suggestion
  height: 50px;
  padding: 1em;
```

**Review by archishadatta — CHANGES_REQUESTED** [review]

**Review by reginawang3495 — CHANGES_REQUESTED** [review]
oops!! I forgot to submit the review :)

**Comment by ldalton02** [comment]
My bad on late response time. I changed Stats.tsx to create an object out of the props and then map the object to display all of the stats. Also moved the random number generating function to common/math.js in the src folder

**Review by reginawang3495 — CHANGES_REQUESTED** [review]
I dmed some things to consider changing as well!!

**Review by reginawang3495 — COMMENTED** [review]
Quick glance — shape of PR looks really great to me!!

**Review by archishadatta — APPROVED** [review]

File: src/components/shared/Stats.tsx [hunk 3]
```diff
@@ -1,13 +1,29 @@
 import '../../styles/stats.scss';
+
+type statsObject = {
```
Comment (reginawang3495) [inline src/components/shared/Stats.tsx:3]: I would make a playerStats object somewhere and use it both here and in the state of App.tsx

File: src/components/shared/Stats.tsx [hunk 4]
```diff
@@ -1,13 +1,29 @@
...
 interface statsProps {
-  battery: number;
+  playerStats: statsObject,
 }
 
-export default function Stats(props:statsProps): JSX.Element {
+export default function Stats(props: statsProps): JSX.Element {
+  const { playerStats } = props;
+
+  const renderStats = () => {
```
Comment (reginawang3495) [inline src/components/shared/Stats.tsx:14]: Since there isn’t much code and this is only used once, I’d prefer to in-line this

**Review by reginawang3495 — CHANGES_REQUESTED** [review]
A couple small ideas to improve + would recommend running “yarn lint-fix”

**Review by reginawang3495 — APPROVED** [review]
looks great!! a few tiny changes, then feel free to merge :)

some small formatting issues (like extra lines, and not having a new line at the end of a file), but not a big deal. Feel free to ask why as always!

File: src/common/types.ts [hunk 5]
```diff
@@ -0,0 +1,3 @@
+export type statsObject = {
+  [key: string]: number,
+}
```
[the remark this answers is not in the document]
Comment (ldalton02) [reply src/common/types.ts:3]: It might not always be the same in the future but good to manually define it to prevent issues

**Pull request merged**